app/presenters/person_presenter.rb

```diff
@@ -30,10 +30,37 @@ def has_policy_responsibilities?
   end
 
   def announcements
-    announcements =
-      model.published_speeches.with_translations(I18n.locale).limit(10).map { |s| SpeechPresenter.new(s, context) } +
-      model.published_news_articles.with_translations(I18n.locale).limit(10).map { |na| NewsArticlePresenter.new(na, context) }
-    announcements.sort_by { |a| a.public_timestamp.to_datetime }.reverse[0..9]
+    search_results = Whitehall.search_client.search(
```

Comment:
I can see the old code is filtering by locale and speeches/news, but I can't see the same here. Is it implicit somehow?